app/javascript/packages/document-capture/context/failed-capture-attempts.tsx

```diff
@@ -0,0 +1,102 @@
+import { createContext, useState } from 'react';
```

Comment:
Is this file meant to be included?

Reply:
It is, sort of, but I made the jsx -> tsx file rename more explicit now in https://github.com/18F/identity-idp/pull/6769/commits/6d0a37729421187bb13137bddb30d63ff6a8bd9d